### PR TITLE
Implement dry saves

### PR DESCRIPTION
**Pull request by Learath2**
I'm not sure how much I like this but it should be helpful

### CODE REVIEW

File: src/game/ddracecommands.h [hunk 1]
```diff
@@ -52,6 +52,7 @@ CONSOLE_COMMAND("unmute", "v[id]", CFGFLAG_SERVER, ConUnmute, this, "")
 CONSOLE_COMMAND("mutes", "", CFGFLAG_SERVER, ConMutes, this, "")
 CONSOLE_COMMAND("moderate", "", CFGFLAG_SERVER, ConModerate, this, "Enables/disables active moderator mode for the player")
 CONSOLE_COMMAND("vote_no", "", CFGFLAG_SERVER, ConVoteNo, this, "Same as \"vote no\"")
+CONSOLE_COMMAND("dry_save", "", CFGFLAG_SERVER, ConDrySave, this, "Dump the current savestring")
```
Comment (def-) [inline src/game/ddracecommands.h]: maybe rather save_dry

File: src/game/server/ddracecommands.cpp [hunk 2]
```diff
@@ -705,3 +706,44 @@ void CGameContext::ConVoteNo(IConsole::IResult *pResult, void *pUserData)
...
+	if(!pPlayer || pSelf->Server()->GetAuthedState(pResult->m_ClientID) != AUTHED_ADMIN)
+		return;
+
+
+	CSaveTeam SavedTeam(pSelf->m_pController);
+	int Num = SavedTeam.save(pPlayer->GetTeam());
+	switch (Num)
+	{
+		case 1:
+			pSelf->SendChatTarget(pResult->m_ClientID, "You have to be in a team (from 1-63)");
```
Comment (def-) [inline src/game/server/ddracecommands.cpp]: I guess something here is copy-pasted. Could refactor it so you only need to write these once?

Comment (Learath2) [reply src/game/server/ddracecommands.cpp]: You mean the entire switch stmt?

File: src/game/server/ddracecommands.cpp [hunk 3]
```diff
@@ -705,3 +706,32 @@ void CGameContext::ConVoteNo(IConsole::IResult *pResult, void *pUserData)
...
+	if(!pPlayer || pSelf->Server()->GetAuthedState(pResult->m_ClientID) != AUTHED_ADMIN)
+		return;
+
+
+	CSaveTeam SavedTeam(pSelf->m_pController);
+	int Result = SavedTeam.save(pPlayer->GetTeam());
+	if(CSaveTeam::HandleSaveError(Result, pResult->m_ClientID, pSelf))
+		return;
+
+	if(!Result)
```
Comment (def-) [inline src/game/server/ddracecommands.cpp]: this is probably not required

File: src/game/server/ddracecommands.cpp [hunk 4]
```diff
@@ -705,3 +706,32 @@ void CGameContext::ConVoteNo(IConsole::IResult *pResult, void *pUserData)
...
+
+void CGameContext::ConDrySave(IConsole::IResult *pResult, void *pUserData)
+{
+	CGameContext *pSelf = (CGameContext *)pUserData;
+
+	CPlayer *pPlayer = pSelf->m_apPlayers[pResult->m_ClientID];
+
+	if(!pPlayer || pSelf->Server()->GetAuthedState(pResult->m_ClientID) != AUTHED_ADMIN)
+		return;
+
```
Comment (def-) [inline src/game/server/ddracecommands.cpp]: one empty line too much

File: src/game/server/ddracecommands.cpp (hunk 3 above)
Comment (Learath2) [reply src/game/server/ddracecommands.cpp]: As long as the handler never falls out of sync with save, but I guess the next rewrite on this file will be an entire cleanup

**Pull request merged**